components/tel/Core/Src/radio.c

```diff
@@ -81,6 +135,14 @@ void RADIO_tx_CAN_msg(CAN_Radio_msg_t *tx_CAN_msg)
   radio_buffer[CAN_CARRIAGE_RETURN_INDEX] = '\r';                                            /* CARRIAGE RETURN */
   radio_buffer[CAN_NEW_LINE_INDEX] = '\n';                                                   /* NEW LINE */
 
-  HAL_UART_Transmit(&huart1, radio_buffer, sizeof(radio_buffer), CAN_TRANSMIT_TIMEOUT);                     /* Transmit over UART */
-}   
+  RADIO_tx_API_Accumulator(radio_buffer);
```

Comment:
@AarjavJain101 Is working on the diagnostics API which will also introduce a UART API which can be used for sending out  radio messages. So the abstraction would look something like: XBee_API_Accumulator --> UART API --> Does uart transmit